src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -42,7 +43,7 @@ export class Eoy2018ArticleHeader extends React.Component<{
 
       return (
         <GridItem key={i} width={[1 / 2, 1 / 3, 1 / 4]} isVisible={isVisible}>
-          <Img src={src} />
+          <Img src={!isTest ? src : ""} />
```

Comment:
This is kind of weird, but was the only way I could think of to hide image urls during tests -- wanted to have a snapshot so we can track changes to all features over time, but this component generates a random order for images on every mount.